src/main/java/kg/jarkyn/cobspecserver/Response.java

```diff
@@ -0,0 +1,13 @@
+package kg.jarkyn.cobspecserver;
+
+public class Response {
+    private String status;
+
+    public Response(String status) {
+        this.status = status;
+    }
+
+    public byte[] getBytes() {
```

Comment:
Not necessarily. But `Response` is a high level abstraction from the very concrete way responses are sent through the wire, isn't it? For us humans a string representation is fine because that's what it is right? Lines of text.
That this high level abstraction suddenly provides information about a byte array is surprising.
But ask yourself _who_ actually does need to know that a byte array is needed? Is it really the `Response` itself? Or another thing that uses a response and wants to send it through a stream?